ui_testing/testcases/basic_tests/test05_contacts.py

```diff
@@ -140,28 +140,31 @@ def test_006_download_contact_sheet(self):
                 self.assertIn(item, fixed_sheet_row_data)
 
     def test_007_create_contact_with_person(self):
+        """
+        New: Contact: Creation Approach: I can create new contact successfully with contact person
+        LIMS-6386
+        """
         contact_data = self.contact_page.create_update_contact()
-
-        self.base_selenium.LOGGER.info('filter by contact no.: {} to get the record'.format(contact_data['Contact No']))
+        self.info('filter by contact no.: {} to get the record'.format(contact_data['Contact No']))
         self.base_selenium.refresh()
 
         contact_record = self.contact_page.search(value=contact_data['Contact No'])[0]
-        self.base_selenium.LOGGER.info('open the record in edit to compare the data')
+        self.contacts_page.info('open the record in edit to compare the data')
         self.contact_page.open_edit_page(row=contact_record)
```

Comment:
@ghada-37 Try this and let me know
```suggestion
        self.contact_page.open_edit_page_by_css_selector(row=contact_record)
```